src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureMethod.cs

```diff
@@ -170,7 +170,7 @@ internal override int CalculateLocalSyntaxOffset(int localPosition, SyntaxTree l
             return _topLevelMethod.CalculateLocalSyntaxOffset(localPosition, localTree);
         }
 
-        public override bool AreLocalsZeroed => _topLevelMethod.AreLocalsZeroed;
+        public sealed override bool AreLocalsZeroed => _topLevelMethod.AreLocalsZeroed;
```

Comment:
>_topLevelMethod [](start = 55, length = 15)

Should we delegate to BaseMethod instead, so that we properly handle the chain of nested local functions?

Reply:
I think you have a point. It might be a problem when we enable attributes on local functions.

Reply:
I believe it's not worth it right now, since it's for a future feature. Delegating it to BaseMethod makes us hit unreachable code, and it would require some debugging and other changes.

Reply:
Specifically, when we debugged through this there was a problem where the LocalFunction was at the top level of a script, where the ContainingSymbol was the Script type, but the `_topLevelMethod` was the `Initialize` method of the script. We probably want to decide whether this is the right behavior in general, which seems out of scope of this PR.